services/profile.ts

```diff
@@ -0,0 +1,115 @@
+import { $fetch, FetchError } from 'ofetch'
+
+const hostToServiceMap = {
+  'kodadot.xyz': 'https://profile.kodadot.workers.dev/',
+  'canary.kodadot.xyz': 'https://profile-beta.kodadot.workers.dev/',
+  'localhost:9090': 'http://localhost:8787',
```

Comment:
It should exist only during the development of this feature.

Reply:
fixed